tests/client/test_orion_client.py

```diff
@@ -732,6 +732,26 @@ async def test_read_nonexistent_concurrency_limit_by_tag(orion_client):
         await orion_client.read_concurrency_limit_by_tag("not-a-real-tag")
 
 
+async def test_resetting_concurrency_limits(orion_client):
+    cl = await orion_client.create_concurrency_limit(
+        tag="an-unimportant-limit", concurrency_limit=100
+    )
+
+    await orion_client.reset_concurrency_limit_by_tag(
+        "an-unimportant-limit", slot_override=[uuid4(), uuid4(), uuid4()]
+    )
+    first_lookup = await orion_client.read_concurrency_limit_by_tag(
+        "an-unimportant-limit"
+    )
+    assert len(first_lookup.active_slots) == 3
+
+    await orion_client.reset_concurrency_limit_by_tag("an-unimportant-limit")
```

Comment:
Should these be separated? I don't feel strongly about this one and am more interested out of curiosity about test design.

Reply:
There wasn't a good way to inject active slots, so we're using the `reset` utility to both show we can customize the active concurrency slots and clear them